libyul/optimiser/Metrics.cpp

```diff
@@ -70,8 +70,14 @@ size_t CodeWeights::costOf(Expression const& _expression) const
 		return functionCallCost;
 	else if (holds_alternative<Identifier>(_expression))
 		return identifierCost;
-	else if (holds_alternative<Literal>(_expression))
-		return literalCost;
+	else if (Literal const* literal = get_if<Literal>(&_expression))
```

Comment:
Can we merge this metric change separately and see if any step/test would behave differently? I guess there's no blocker merging this metric change already?

Reply:
Done in #11288.